x/distribution/types/msg.go

```diff
@@ -34,15 +33,6 @@ func NewMsgWithdrawValidatorCommission(valAddr string) *MsgWithdrawValidatorComm
 	}
 }
 
-// NewMsgFundCommunityPool returns a new MsgFundCommunityPool with a sender and
-// a funding amount.
-func NewMsgFundCommunityPool(amount sdk.Coins, depositor string) *MsgFundCommunityPool {
```

Comment:
Do you think it's worth it to have some kind of legacy support? Still support the previous message and it converts it to a pool message. Like there was for gov v1beta1 -> v1.
This is quite breaking otherwise, this message was just introduce in v0.47.

Personally I'd vote for some kind of legacy support and have the message server do the conversion (like gov).

Reply:
the distribution module does have the msg server to do the conversion and for this `NewMsgFundCommunityPool`, it is not really used anywhere other than CLI cmd and simulation operations which are removed in this PR

Reply:
if the router is present in distr then we might as well add it

Reply:
cc @likhita-809